plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.scss

```diff
@@ -9,21 +7,135 @@
 		width: 100%;
 	}
 
+	.settings-payment-providers__header-title {
+		font-size: 16px;
```

Comment:
maybe we should use some aliases according to the guidelines to avoid dupliacting these styles, like `@include subtitle` or something like that?

Reply:
will address in design issue https://github.com/woocommerce/woocommerce/issues/52113